docs/contribute.rst

```diff
@@ -116,7 +71,6 @@ for your own :doc:`custom installation <custom_installs/index>`.
 .. toctree::
    :maxdepth: 1
 
-   development/install
    development/standards
```

Comment:
I still don't quite understand how I'm meant to get to this TOC. It's hidden in a second-level page with the title "Contributing" and then not even in view when you load the page. I'd really like to at least get some of this content onto the main TOC and front page of the docs. 

Currently searching for "install" or "develop" doesn't lead to this content when hitting the front-page of the site, which I think we should likely address here.

Reply:
Of note, this is also a weird place for this TOC -- it should live in the `development/index.rst`, not inside a `contribute.rst` at the top-level. I think linking to that index and adding it to the top-level TOC would solve a lot of my issue here.

Reply:
I am putting together a PR with my suggestions.

Reply:
https://github.com/readthedocs/readthedocs.org/pull/6861